libs/scripts/scripts.js

```diff
@@ -22,11 +22,20 @@ import locales from '../utils/locales.js';
 const prodDomains = ['milo.adobe.com'];
 
 const stageDomainsMap = {
-  'www.adobe.com': 'www.stage.adobe.com',
-  'blog.adobe.com': 'blog.stage.adobe.com',
-  'business.adobe.com': 'business.stage.adobe.com',
-  'helpx.adobe.com': 'helpx.stage.adobe.com',
-  'news.adobe.com': 'news.stage.adobe.com',
+  'www.stage.adobe.com': { 'www.adobe.com': 'www.stage.adobe.com' },
+  'business.stage.adobe.com': {
+    'www.adobe.com': 'www.stage.adobe.com',
+    'business.adobe.com': 'business.stage.adobe.com',
+    'blog.adobe.com': 'blog.stage.adobe.com',
+    'helpx.adobe.com': 'helpx.stage.adobe.com',
+    'news.adobe.com': 'news.stage.adobe.com',
+  },
+  '--milo--robert-bogos.hlx.page': {
```

Comment:
This seems a bit self-centered 😛 I'd guess this shouldn't be in the final code?

Reply:
I will remove it 😂, is just to make QA's work easier